sdk/eventhub/event-hubs/changelog.md

```diff
@@ -1,3 +1,26 @@
+### 2019-08-06 5.0.0-preview.2
+
+#### General
+- The sender is refactored to avoid the `TooManyEventListeners` warning which would occur before if too many send requests were in flight at the same time from the same sender.
```

Comment:
```suggestion
- The sender is refactored to avoid the warning around too may listeners being attached which would occur before if too many send requests were in flight at the same time from the same sender.
```